### ISSUE TITLE
Part-solid pulmonary nodule

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6636]
**Preferred term label:**Part-solid pulmonary nodule

**Synonyms**
PSN

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Subsolid pulmonary nodule (once created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik 
adding term
adding the synonym 
Part-solid ground-glass nodule
